bindings/nodejs/src/lib.rs

```diff
@@ -1,70 +1,65 @@
 // Copyright 2023 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-mod client;
-mod secret_manager;
-mod wallet;
+pub mod client;
+pub mod secret_manager;
+pub mod wallet;
 
 use iota_sdk_bindings_core::{
     call_utils_method as rust_call_utils_method, init_logger as rust_init_logger, Response, UtilsMethod,
 };
-use neon::prelude::*;
-use once_cell::sync::Lazy;
-use tokio::runtime::Runtime;
+use napi::{Error, Result, Status};
+use napi_derive::napi;
 
-pub static RUNTIME: Lazy<Runtime> = Lazy::new(|| Runtime::new().unwrap());
+#[derive(Debug, thiserror::Error)]
+#[non_exhaustive]
+pub enum NodejsError {
+    /// Bindings errors.
+    #[error(transparent)]
+    Bindings(#[from] iota_sdk_bindings_core::Error),
```

Comment:
I think it's a bit weird as a variant name, knowing we're already inside a bindings module, maybe Core ?

Reply:
BindingsCore or just Core?

Reply:
I'd personally go with Core

Reply:
Renamed